src/app/page/Donate/DonationPage.jsx

```diff
@@ -147,9 +148,9 @@ function SuccessState({ classes, state }) {
           <Divider className={classes.marginVertical} />
           <Typography align="left" variant="subtitle2" gutterBottom>
             <em>
-              {org.name} is a registered 501(c)3 non-profit organization #{org.EINNumber}. Your
-              donation is tax deductible to the extent allowable by law. No goods or services were
-              provided by {org.name} in return for this contribution.
+              {org.displayName} is a registered 501(c)3 non-profit organization #{org.EINNumber}.
```

Comment:
fwiw this has to get commented out for now anyways.. waiting to hear back formally but pretty sure it will be
> tax deductible yes 
> registered 501(c)3  no


 at least they will be looking for a fiscal sponsorship..

maybe this whole thing could be in an 'if' block based on existence of EIN number.. 

that said you don't have to address this comment in this PR

Reply:
This should only show when the org actually has an EIN number set